mono/config.py

```diff
@@ -11,14 +11,24 @@ def configure(env):
     env.use_ptrcall = True
 
     if env['platform'] == 'windows':
-        if env['bits'] == '64':
-            env.Append(LIBPATH = 'C:\\Program Files\\Mono\\lib\\')
-            env.Append(CPPPATH = 'C:\\Program Files\\Mono\\include\\mono-2.0\\')
+        if env['bits'] == '32':
+            if os.getenv('MONO32_PREFIX'):
+                mono_path = os.getenv('MONO32_PREFIX')
+            else:
+                mono_path = os.path.join(os.getenv('PROGRAMFILES(X86)') if os.getenv('PROGRAMFILES(X86)') else os.getenv('PROGRAMFILES'), 'Mono')
         else:
-            env.Append(LIBPATH = 'C:\\Program Files (x86)\\Mono\\lib\\')
-            env.Append(CPPPATH = 'C:\\Program Files (x86)\\Mono\\include\\mono-2.0\\')
+            if os.getenv('MONO64_PREFIX'):
+                mono_path = os.getenv('MONO64_PREFIX')
+            else:
+                mono_path = os.path.join(os.getenv('PROGRAMW6432'), 'Mono')
 
-        env.Append(LINKFLAGS = [ 'monosgen-2.0.lib' ])
+        env.Append(LIBPATH = os.path.join(mono_path, 'lib'))
+        env.Append(CPPPATH = os.path.join(mono_path, 'include', 'mono-2.0'))
+
+        if (os.getenv("VCINSTALLDIR")):
+            env.Append(LINKFLAGS = 'monosgen-2.0' + Environment()['LIBSUFFIX'])
```

Comment:
Godot overrides the `LIBSUFFIX` variable, that's why must manually append it to `LINKFLAGS` instead of using `LIBS`.